aiohttp/web.py

```diff
@@ -81,6 +85,21 @@
 ]
 
 
+class WebSocketClosed(DisconnectedError):
```

Comment:
i think this should be moved to errors.py, also base class should be ClientDisconnecte